src/bin/release-please.ts

```diff
@@ -71,12 +71,11 @@ export const parser = yargs
       releaseType(yargs, 'node');
     },
     (argv: ReleasePRFactoryOptions) => {
-      const releasePR = factory.releasePR(argv);
-      releasePR
-        .latestTag()
+      factory
+        .runCommand('latest-tag', argv)
         .catch(handleError)
         .then(latestTag => {
-          console.log(latestTag);
+          if (latestTag) console.log(latestTag);
```

Comment:
maybe in the `else` we could say `no latest tag found`? might also be worth setting `process.exitCode = 1`?